lib/ClangImporter/ClangImporter.cpp

```diff
@@ -533,6 +533,12 @@ getNormalInvocationArguments(std::vector<std::string> &invocationArgStrs,
       });
     }
   } else {
+    invocationArgStrs.insert(invocationArgStrs.end(), {
+      // Most of the good portable functions used in the Glibc module are hidden
+      // behind this.
+      "-D_GNU_SOURCE",
```

Comment:
Is it possible to write a test for this? (Not that the macro is defined, but that we actually get one of the "good portable functions" that was previously hidden.)

Reply:
sure yeah. getline is a really common one that goes missing without a feature define. Also could do dladdr, dlsym, lseek64, or getaddrinfo_a.

Reply:
done. added a LibcFeatureFlag.swift test in the last commit.